flake.nix

```
@@ -0,0 +1,35 @@
+{
+  description = "Fuzzy finder for Makefile";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+    utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system: let
+      pkgs = nixpkgs.legacyPackages.${system};
+        in
+      {
+        devShells.default = pkgs.mkShell {
+          packages = with pkgs; [
+            bat
+            cargo
+            gnumake
+          ];
+        };
+
+        packages = rec {
+          fzf-make = pkgs.rustPlatform.buildRustPackage {
+            pname = "fzf-make";
+            version = "0.6.0";
```

Sorry about this too.
I released `v0.7.0` after you created this PR. 🙇 
So please change as follows.

```suggestion
            version = "0.7.0";
```